operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -125,6 +125,32 @@ public class KafkaCluster extends AbstractKafkaCluster {
     public static final String MAX_PARTITIONS = "max.partitions";
     public static final String MESSAGE_MAX_BYTES = "message.max.bytes";
 
+    private static final String ENFORCED_RULE_CONFIG = String.join(",",
```

Comment:
I don't think we want this hardcoded in the Java code.  You should follow the pattern of putting it in the application.properties (common to all), standard.properties (used by standard( and developer.properties (used by developer).

That will also allow us to to override the settings from environment variables, should the need ever arise.

Reply:
Thanks @k-wall for the comments. I'm working on it, and has a question:
After we move the setting to `strandard.properties` and `developer.properties`. The reason is we'll set the enforce rule for standard to `min.insync.replicas:2`, but there is a feature allows user to scale down the cluster, so the `min.insync.replicas` will change to 1.  But the enforce rule doesn't change, so it'll fail.

https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/blob/0.21.x/operator/src/test/java/org/bf2/operator/operands/KafkaClusterTest.java#L257-L260

Since we'll dynamically set `min.insync.replicas` value based on the `scaling-and-replication-factor` property, I think we should also apply the same pattern to `enforce rule`. That is, in enforce rule config, we don't set `min.insync.replicas`, ex:
`kafka.topic-config-enforced-rule="index.interval.bytes:4096,compression.type:producer"`
And during setting the kakfa broker config, we dynamically append "expected min.insync.replicas" value to it, just like how we set kafka broker `min.insync.replicas` value.


WDYT?
I've updated the PR with this solution. Let me know if you have other suggestion. Thank you.